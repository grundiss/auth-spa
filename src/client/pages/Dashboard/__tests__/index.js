import * as React from "react";
import { render } from "enzyme";
import { Dashboard } from "../index";

jest.unmock("../index");
jest.mock("@material-ui/core/Button", () => ({ children }) => (
  <div data-mockof="@material-ui/core/Button">{children}</div>
));

describe("pages / Dashboard", function() {
  it("renders", function() {
    expect(render(<Dashboard />)).toMatchSnapshot();
  });
});
