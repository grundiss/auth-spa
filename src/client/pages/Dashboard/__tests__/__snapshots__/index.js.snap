// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages / Dashboard renders 1`] = `
<div
  data-mockof="client/AuthenticationContex"
>
  <div>
    <h1>
      Hello, Mr. Mocker McMockface
    </h1>
    <div>
      <div
        data-mockof="@material-ui/core/Button"
      >
        Logout
      </div>
    </div>
  </div>
</div>
`;
